content/docs/user-guide/experiment-management/hydra-composition.md

```diff
@@ -2,37 +2,46 @@
 
 _New in DVC 2.25.0 (see `dvc version`)_
 
-In this guide, we explain how to use [Hydra](https://hydra.cc/) composition for
-configuring DVC <abbr>experiments</abbr>.
+[Hydra](https://hydra.cc/) is a framework to configure complex applications. DVC
+supports Hydra's [config composition] as a way to configure [experiment runs].
 
-## How it works
+<admon type="info">
 
-On each `dvc exp run` call, DVC will use Hydra to **compose** a single
-configuration object and **dump** it to `params.yaml`. This will happen
-**before** the experiment starts running.
+At the moment you must explicitly enable this feature with:
+
+```dvc
+$ dvc config hydra.enabled True
+```
 
-This allows to combine Hydra composition and DVC <abbr>parameters</abbr> to
-configure DVC <abbr>pipelines</abbr>. DVC pipelines can run multiple steps of
-different shell commands, instead of a single Python script, and use features
-like [Templating](/doc/user-guide/project-structure/dvcyaml-files#templating)
-and [`foreach`](/doc/user-guide/project-structure/dvcyaml-files#foreach).
+</admon>
 
-## Setting up Hydra
+[config composition]:
+  https://hydra.cc/docs/tutorials/basic/your_first_app/composition/
+[experiment runs]: /doc/user-guide/experiment-management/running-experiments
+
+## How it works
+
+Upon `dvc exp run`, Hydra will be used to compose a single `params.yaml` before
+executing the experiment. Hydra config and existing <abbr>parameters</abbr> will
+be combined to configure the [underlying DVC pipeline](#running-experiments)
+that contains your experiment.
```

Comment:
```suggestion
Upon `dvc exp run`, Hydra will be used to compose a single `params.yaml` before
executing the experiment. The composed `params.yaml` will
be used to configure the [underlying DVC pipeline](#running-experiments)
that contains your experiment.
```

Minor: `Hydra config and existing parameters will be combined` seems a bit vague and confusing. How are they combined?

Reply:
Hmmm yeah I guess I though existing manual params.yaml files would also be taken into consideration, but this will completely overwrite those right? E.g. if you have a manual params.yaml file but later enable Hydra and setup its config groups and a default list.

Going with this for now:
```suggestion
Upon `dvc exp run`, Hydra will be used to compose a single `params.yaml` before
executing the experiment. Its <abbr>parameters</abbr> will configure the
[underlying DVC pipeline](#running-experiments) that contains your experiment.
```

Reply:
> this will completely overwrite those right?

Just to confirm, that's correct.